lib/intercom_stats/intercom/worker.ex

```diff
@@ -110,8 +122,6 @@ defmodule IntercomStats.Intercom.Worker do
       item
       |> Map.put("company_name", retrieve_company_name(conversation))
       |> Map.put("tags", retrieve_tags_for_conversation(conversation))
-
-    if Enum.any?(item_with_tags["tags"]) do
       item_with_tags
```

Comment:
You define `item_with_tags` twice now, what was the intention of removing this `if` statement?

Reply:
The if statement enumerates only those files with tags, but I wanted to include items without tags. I guess item_with_tags isn't needed anymore at all here right?